### PR TITLE
Issue 1 user can search for games

### PR DESCRIPTION
**Pull request by c-alvin**
mobileissue1
desktopissue1

### CODE REVIEW

**Comment by c-alvin** [comment]
image

File: server/index.js [hunk 1]
```diff
@@ -2,13 +2,26 @@ require('dotenv/config');
...
-app.get('/api/hello', (req, res) => {
-  res.json({ hello: 'world' });
+app.get('/api/search', (req, res) => {
+  // console.log(req.query.term);
+  const search = req.query.term;
+  fetch('https://api.igdb.com/v4/games', {
+    method: 'POST',
+    headers: {
+      'Client-ID': process.env.CLIENT_ID,
+      Authorization: 'Bearer wb9acv3ulq4tsa3zc27zkli61edii0'
```
Comment (thebearingedge) [inline server/index.js:18]: You should pull this out into .env. Never commit keys like this.

File: server/index.js [hunk 2]
```diff
@@ -2,13 +2,26 @@ require('dotenv/config');
...
+    },
+    body: `fields cover.*,first_release_date, release_dates.*, screenshots.*, name, platforms.*; search: "${search}"; limit 10; offset 0;`
+  })
+    .then(res => res.json())
+    .then(data => res.status(201).json(data))
+    .catch(err => console.error(err));
```
Comment (thebearingedge) [inline server/index.js:24]: We always pass errors to `next()` in express apps.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, just some minor cleanup, please scan all of your files for commented out lines

https://github.com/c-alvin/final-project/pull/11/files#

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**